board_signature/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: 2025 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+{
+    "name": "Board signature",
+    "summary": "Adds board representative's name, function and signature to companies",
+    "version": "16.0.1.0.0",
+    "category": "Cooperative management",
```

Comment:
I didn’t see first time but here we can change the category.

```suggestion
    "category": "Reporting",
```